javascript/javascript1/week4/index.js

```diff
@@ -0,0 +1,115 @@
+
+let nameArray=[]
+let todoArray=[]
+let todoName=[]
+function getReply(command) {   
+
+    if (command.includes('Hello my name is Benjamin'))
+    {
+        const splitCommand = command.split(' ')
+        nameArray.push(splitCommand[4])
+        console.log(`Nice to meet you ${nameArray[0]}`)
+    }
+          
+   if(command.includes('What is my name')){
+    if (nameArray!="")
+    console.log(`Your name is ${nameArray[0]}`)
+  else 
+    console.log("Your name has not yet been mentioned") 
+    
+   }  
+   
+   
+    if (command.includes('Add fishing to my todo')){
+     
+        todoName.push('fishing')
+        console.log('fishing added to your todo')              
+    }else if (command.includes('Add singing in the shower to my todo')){
+     
+        todoName.push('singing in the shower')
+        console.log('singing in the shower added to your todo')              
+    }
+
+    if(command.includes('Remove fishing from my todo'))
+    {
+        todoName.shift()
```

Comment:
what if we don't have fishing on the to-do list or fishing wasn't the first element of the array. I think it's better to find the fishing index in an array and then remove it or use the filter.